extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -93,7 +93,7 @@ class AmpSocialShare extends AMP.BaseElement {
       let container = elementByTag(this.element, 'span');
       if (container == null) {
         container = this.getWin().document.createElement('span');
-        container.classList.add(this.type_);
+        container.classList.add('amp-social-share-' + this.type_);
```

Comment:
add documentation in the md file to reflect what the naming constraints are - and how ppl are expected to style them.